src/plugins/graph/components/graph.tsx

```diff
@@ -143,7 +143,9 @@ export const Graph = observer(
     }
   };
 
-  useDataTips({dotsRef, dataset, graphModel, enableAnimation});
+  // useDataTips({dotsRef, dataset, graphModel, enableAnimation});
+  //useDataTips hook is used to identify individual points in a dense scatterplot
+  //it should be commented out for now as it shrinks outer circle when hovered over, but may prove useful in the future
```

Comment:
@lbondaryk confirmed recently that she's going to want this feature to work at some point.

Reply:
Gotcha - so then I'll just leave it "as is" - commented out.